README.md

```diff
@@ -45,8 +52,16 @@ curl https://cdn.rawgit.com/pimterry/notes/v0.2.0/notes.bash_completion > /usr/s
 
 You'll need to open a new shell for this to take effect.
 
+## What if I want to uninstall this?
+If you used the automated install script to install notes, you can uninstall it the same way by running:
+```bash
+export uninstall=1 && curl https://rawgit.com/pimterry/notes/master/install.sh | bash
```

Comment:
Because of the sudo env vars thing, and because this is a little unorthodox, I think we should probably simplify, and just pass `uninstall` as a command line argument instead. `... | bash -s uninstall` should work fine for piping, and being able to manually pull it down and then run just `./install.sh uninstall` feels nice and clear.